pkg/agent/attestor/node/node.go

```diff
@@ -70,12 +71,21 @@ func (a *attestor) Attest(ctx context.Context) (res *AttestationResult, err erro
 		return nil, err
 	}
 
-	if svid == nil {
+	switch {
+	case svid == nil:
 		svid, bundle, err = a.newSVID(ctx, key, bundle)
 		if err != nil {
 			return nil, err
 		}
+	case bootstrapping:
+		// This is a bizarre case where we have an SVID but were unable to
+		// load a bundle from the cache which suggests some tampering with the
+		// cache on disk.
+		// TODO: figure out if there are safe actions to take instead of
+		// failing out
+		return nil, errs.New("SVID loaded but no bundle in cache")
```

Comment:
`loadBundle` has no knowledge of whether or not the SVID will be successfully loaded. `loadSVID` has no knowledge of whether or not the bundle has successfully loaded. The situation we want to detect is one where we think we are bootstrapping (because there is no bundle cached) but there is an SVID persisted. In this situation we have to assume some sort of cache inconsistency. We could decide to 1) fail out (what I chose), 2) force re-attestation (by only trying to load the SVID if we have a bundle). Thoughts?